_config.yml

```diff
@@ -59,6 +59,7 @@ tkui: "Telerik Kendo UI"
 tuin: "Telerik UI for NativeScript"
 
 # Telerik UI services
+ab-sab: "Code (AppBuilder)"
```

Comment:
Do we need to tie the two? We can just use {{site.ab-s}} ({{site.ab}}) in the text.